src/App.jsx

```diff
@@ -14,57 +16,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selGood, setSelecGood] = useState(
+    goods.find(good => good === 'Jam') || goods[0],
+  );
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selGood ? `${selGood} is selected` : `No goods selected`}
+        {selGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelecGood('')}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            return (
+              <tr
+                key={good}
```

Comment:
Using the 'good' value, which is a string from the array, as a key in a map function can lead to issues if the list contains non-unique values. It's better to use a unique identifier as a key, such as an ID.